ucp/useraccounts_module.php

```diff
@@ -464,15 +429,15 @@ function main($id, $mode)
 						trigger_error($this->language->lang('FORM_INVALID') . '<br><br>' . $this->user->lang('RETURN_UCP', '<a href="' . $this->u_action . '">', '</a>'));
 					}
 
-					// Get current item order value
-					$sql = "SELECT badge_order as current_order
+					// Get current item order value.
+					$sql = "SELECT badge_order AS current_order
 						FROM $user_badge_table
-						WHERE item_id = $item_id"; // No need to check for user_id, item_id is the table's primary key.
+						WHERE item_id = $item_id";
```

Comment:
Please use `WHERE item_id  = ' . (int)$item_id` to make it clear that this is safe.